openshift/ci-operator/generate-dockerfiles.sh

```diff
@@ -5,17 +5,18 @@ set -e
 function generate_dockerfiles() {
   local target_dir=$1; shift
   # Remove old images and re-generate, avoid stale images hanging around.
-  for img in $@; do
-    local image_base=$(basename $img)
+  for img in "$@"; do
+    local image_base
+    image_base=$(basename "$img")
     local kodata_path="$img/kodata"
-    mkdir -p $target_dir/$image_base
+    mkdir -p "$target_dir"/"$image_base"
```

Comment:
I see we don't have consistency about using or not using brackets in vars :thinking

Reply:
Yeah, seems to be the "recommendation" to use them, thus the linter default IMHO.